src/engine/Engine.ts

```diff
@@ -971,6 +972,11 @@ module ex {
       private _update(delta: number) {
          if (this._isLoading) {
             // suspend updates untill loading is finished
+            this._loader.update(this, delta);
+            // Update input listeners
+            this.input.keyboard.update(delta);
```

Comment:
This is ugly but because Loader is not part of a Scene (hint hint), the input never gets cleared and results in bad input handling for the tap to play trigger. I didn't want to move the lines below above because that could cause issues (frame input gets cleared before actors can use it).